perception/map_based_prediction/README.md

```diff
@@ -210,21 +210,21 @@ If the target object is inside the road or crosswalk, this module outputs one or
 
 ### Input
 
-| Name                                                     | Type                                                 | Description                                                |
-| -------------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------------- |
-| `~/perception/object_recognition/tracking/objects`       | `autoware_auto_perception_msgs::msg::TrackedObjects` | tracking objects without predicted path.                   |
-| `~/vector_map`                                           | `autoware_auto_mapping_msgs::msg::HADMapBin`         | binary data of Lanelet2 Map.                               |
-| '~/perception/traffic_light_recognition/traffic_signals' | 'autoware_perception_msgs::msg::TrafficSignalArray;' | rearranged information on the corresponding traffic lights |
+| Name                                                     | Type                                                    | Description                                                |
+| -------------------------------------------------------- | ------------------------------------------------------- | ---------------------------------------------------------- |
+| `~/perception/object_recognition/tracking/objects`       | `autoware_perception_msgs::msg::TrackedObjects`         | tracking objects without predicted path.                   |
+| `~/vector_map`                                           | `autoware_map_msgs::msg::LaneletMapBin`                 | binary data of Lanelet2 Map.                               |
+| `~/perception/traffic_light_recognition/traffic_signals` | `autoware_perception_msgs::msg::TrafficLightGroupArray` | rearranged information on the corresponding traffic lights |
```

Comment:
Note : It may be necessary to change the topic name to traffic_light_group, but first complete the transition to autoware_auto_msgs and then use a separate PR.